frontend/packages/console-app/console-extensions.json

```diff
@@ -65,5 +65,51 @@
     "flags": {
       "required": ["CAN_LIST_VSC"]
     }
+  },
+  {
+    "type": "console.user-settings",
+    "properties": {
+      "id": "console.preferredPerspective",
+      "label": "%console-app~Perspective%",
+      "groupId": "general",
```

Comment:
Can you reorder this and move (all) groups above the `preferredPerspective`? It's easier to understand if the refered groups comes before the reference.